tests/functional-tests/utils/driver.py

```diff
@@ -318,6 +319,45 @@ def create_pvc_from_snapshot(pvc_values, sc_name, pvc_name, snap_name, created_o
         assert False
 
 
+def create_clone_pvc(pvc_values, sc_name, pvc_name, from_pvc_name, created_objects):
```

Comment:
Please, add comments wherever possible how this function is working and parameter/function definition.